### ISSUE TITLE
DataTable `getFormatter not a function` bug

### ISSUE THREAD

**Issue by brendancol** [bokeh/bokeh#4086]
Getting an error when using DataTable:

**Reproduction:**
`python examples/models/data_tables.html`

**Error:**
`Uncaught TypeError: getFormatter(...) is not a function`
`line 1476: slick.grid.js`

Looks like this commit is where it broke:

```
commit 90c6e1bfa5fc962fa151298b6cfc32d8eeca4375
Merge: fa1a222 5ea2fb7
Author: Mateusz Paprocki <mattpap@gmail.com>
Date:   Tue Mar 29 23:55:04 2016 +0200

    Merge pull request #4037 from bokeh/mattpap/ts_api

    Implementation of TypeScript API
```

**Comment by brendancol** [comment]
ping @mattpap

**Comment by mattpap** [comment]
Fixed in #4085.
